CMakeLists.txt

```diff
@@ -114,9 +114,11 @@ function(build_ngraph)
     set(SDL_cmake_included ON)
     # set(NGRAPH_COMPONENT_PREFIX "deployment_tools/ngraph/")
     add_subdirectory(ngraph)
-    set(NGRAPH_LIBRARIES ngraph PARENT_SCOPE)
+    set(NGRAPH_LIBRARIES ngraph common PARENT_SCOPE)
 endfunction()
 
+add_subdirectory(libs)
```

Comment:
may be we can add a folder called `openvino`? It will contain `itt` library?

E.g. we can include this library using:
```cmake 
target_link_libraries(inference_engine PRIVATE openvino_itt)
```
In `cpp` files:
```cpp
#include <opevino/itt.hpp>
```

Reply:
Looks like we also need to export the library somehow for external plugins. Looks like `InferenceEngineDeveloperPackage` is a probable place...

Reply:
Ok, now we need to export this library to use in 3rd party code / external plugins.